docs/notebooks/documents/getting_started.py

```diff
@@ -0,0 +1,159 @@
+# ---
+
+# %% [markdown]
+# # Getting Started
+#
+# This document provides an overview for preparing to work with PDO contracts through the contract
+# docker container. It will walk through the necessary steps to set up a functional
+# configuration. It does assume that the reader is familiar with
+# [basic operation of Jupyter notebooks](https://jupytext.readthedocs.io/en/latest/).
+#
+# %% [markdown]
+# ## How Contracts Use Jupyter
+#
+# Many of the PDO contract families provide Jupyter notebooks to simplify interaction with contract
+# objects. Commonly, the notebooks provide a factory for instantiating a notebook for each new
+# contract from one of the contract specific templates. The templates provided generally include five
+# common sections:
+#
+# * Configure the Contract Object
+# * Initialize the Jupyter Interpreter and the PDO Environment
+# * Initialize the Contract Object Context
+# * Operate on the Contract
+# * View Contract Metadata
+#
+# *Configure the Contract Object*: Each notebook either loads an existing contract object from a
+# file (see the PDO documentation on contract collections) or creates a new contract object based on
+# an initial set of parameters. To create a new contract object, a factory will collect input that
+# is used to instantiate a template notebook with the set of required parameters.
+#
+# *Initialize the Jupyter Interpreter and PDO Environment*: To initialize the Jupyter interpreter,
+# the notebook loads the Juptyer helper module. This module imports all of the relevant PDO and
+# Jupyter IPython modules to simplify code in the notebook. It also defines several procedures that
+# are useful for initializing and interacting with the PDO environment.
+#
+# In addition, the Jupyter interpreter initialization configures an IPython extension that makes it
+# easier to provide code for multiple types of operations that can be performed on the contract
+# object. Specifically, the extension defines a magic keyword, `skip`, that takes a value or
+# expression that, if it evaluates to True, causes the code section to be skipped during notebook
+# execution.
+#
+# *Initialize the Contract Object Context*: The next section in the notebook creates a PDO context
+# for the contract object (see the PDO documentation for more information on contexts).  The context
+# includes all information necessary to create the contract object (and its dependencies) based on
+# the initial set of parameters provided by the factory notebook. Beyond the initial parameters, the
+# context allows for fine-grained customization of the contract object, though the defaults are
+# usually sufficient.
+#
+# *Operate on the Contract*: The contract object may be used once it has been created and
+# initialized. In general, cells in this section of the notebook are turned off by default; that is,
+# `%%skip True` is added at the top of the cell. To perform an operation, change the top line to
+# `%%skip False` and evaluate the cell.
```

Comment:
Same comment as above "Operate on the contract" -> Perhaps change this to "operate on the app"

Reply:
> Same comment as above "Operate on the contract" -> Perhaps change this to "operate on the app"

btw... i really hope you've spent this much time on the other new notebooks out there.

Reply:
while i don't disagree... the metaphor for a notebook (app? contract?) runs through much of our documentation. can i suggest that rather than microanalyze it in one document, that we step back and address the bigger question of terminology (and consistency with it) in an issue? i'm tempted to completely remove this section & leave this document focused on operational configuration.